share/locale/en.json

```diff
@@ -1,165 +1,165 @@
-    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every share will have the same address (to use one-time addresses, disable persistence in Settings)",
-    "gui_url_label_stay_open": "This share will not expire automatically unless a timer is set.",
-    "gui_url_label_onetime": "This share will expire after the first download",
-    "gui_url_label_onetime_and_persistent": "This share will expire after the first download<br><br>Every share will have the same address (to use one-time addresses, disable persistence in the Settings)",
-    "gui_status_indicator_stopped": "Ready to Share",
-    "gui_status_indicator_working": "Starting…",
-    "gui_status_indicator_started": "Sharing",
+    "gui_url_description": "<b>Anyone</b> with this OnionShare address your files using the <b>Tor Browser</b>: <img src='{}' />",
+    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every subsequent share will be available at the same address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_url_label_stay_open": "Add a timer to automatically expire the share.",
+    "gui_url_label_onetime": "This OnionShare expires after one OnionSharing",
+    "gui_url_label_onetime_and_persistent": "This OnionShare expires after completion<br><br>Every subsequent OnionShare will be available at the same OnionShare address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_status_indicator_stopped": "Ready to OnionShare",
+    "gui_status_indicator_working": "Hyping…",
+    "gui_status_indicator_started": "OnionSharing",
     "gui_file_info": "{} Files, {}",
     "gui_file_info_single": "{} File, {}",
-    "info_in_progress_downloads_tooltip": "{} download(s) in progress",
-    "info_completed_downloads_tooltip": "{} download(s) completed",
-    "error_cannot_create_downloads_dir": "Error creating downloads folder: {}",
-    "error_downloads_dir_not_writable": "The downloads folder isn't writable: {}",
-    "receive_mode_downloads_dir": "Files people send you will appear in this folder: {}",
-    "receive_mode_warning": "Warning: Some files can hack your computer if you open them! Only open files from people you trust, or if you know what you're doing.",
+    "info_in_progress_downloads_tooltip": "{} underway",
+    "info_completed_downloads_tooltip": "{} completed",
+    "error_cannot_create_downloads_dir": "Error creating \"OnionShare\" folder: {}",
+    "error_downloads_dir_not_writable": "The \"OnionShare\" folder is write protected: {}",
+    "receive_mode_downloads_dir": "Files sent to you appear in this folder: {}",
+    "receive_mode_warning": "Warning: Some files can take control of your computer if you open them! Only open files from people you trust to know what they are doing.",
```

Comment:
Heh actually I think it should stay "from people you trust, or if you know what you're doing". If I'm trying to hack you by uploading a malicious PDF to your receive mode OnionShare and hope that you open it, I probably know what _I'm_ doing, just you shouldn't open that file unless you really know what _you're_ doing -- or you trust me not to try to hack you.

Basically, OnionShare receive mode is a new channel for phishing attacks. The "don't open email attachments from people you don't trust" applies just as much here.

So I think it should be: "Warning: Some files can take control of your computer if you open them! Only open files from people you trust, or if you know what you're doing."

Reply:
If you don't know what you are doing, you are left trusting everything someone does, by extention of trusting them, personally. The operator here is "or", so that interpretation is valid.

To elaborate on the latter sentence:
Trusting someone to know what they are doing, extends to them uploading a cute PDF to you, but actually it is an executable and whatever else since PDF sucks.